tastyworks/models/watchlists.py

```diff
@@ -0,0 +1,95 @@
+import aiohttp
+from tastyworks.models.session import TastyAPISession
+
+
+class Watchlist(object):
+    def __init__(self, name=None):
+        self.name = name
+        self.group_name = None
+        self.securities = {}
+
+    @classmethod
+    def from_list(cls, data: list):
+        """From List
+
+        Generator for making a watchlist.
+
+        Notes:
+            There is an instability issue with Tastyworks API.  Hence
```

Comment:
Is this for the `instrument-type`/`instrument_type` difference?

If so, can we not abstract this away from the API user (i.e. our customer) and dictate a consistent one (your choice on which one of the two :) ) and we then try both with the tastyworks API transparently?

I think it makes sense to not make tastyworks's inconsistencies, our customers' problem :)

Reply:
I like this approach, I'll update it.  It makes sense to fix consistency for package consumers where we can.